protocol-configuration/collateralizationOracle.ts

```diff
@@ -11,12 +11,19 @@ const collateralizationAddresses: CollateralizationAddressType = {
     'rariTimelockFeiOldLens',
     'tribalCouncilTimelockFeiLens'
   ],
-  lusd: ['lusdPSM'],
-  dai: ['compoundDaiPCVDepositWrapper', 'daiFixedPricePSM', 'ethToDaiLensDai'],
+  lusd: ['lusdPSM', 'lusdHoldingDeposit'],
```

Comment:
shouldn't lusdPSM and ethPSM and voltDepositWrapper and aaveEthPCVDepositWrapper also get removed from CR oracle ?

Reply:
Yep, have removed the `aaveEthPCVDepositWrapper` and deprecated